arangod/RestServer/DatabaseFeature.h

```diff
@@ -140,6 +140,18 @@ class DatabaseFeature : public application_features::ApplicationFeature {
   bool forceSyncProperties() const { return _forceSyncProperties; }
   void forceSyncProperties(bool value) { _forceSyncProperties = value; }
   bool waitForSync() const { return _defaultWaitForSync; }
+  
+  /// @brief whether or not extended names for databases can be used
+  bool extendedNamesForDatabases() const { return _extendedNamesForDatabases; }
```

Comment:
For boolean variables and getters I usually prefer names with is/has/can/use prefix because in most cases this makes the code more expressive. (Just a remark, no need to change this here and now!)

Reply:
I will leave it as it is for now, because we have used a different convention before, and we should agree on some convention to follow as a company.